### DISCUSSION TITLE
[question]  Where is the test suite ?

### DISCUSSION THREAD

**Discussion by bobymicroby** [dani-garcia/vaultwarden#1392] [Ideas]
Guys, I don't see a single line of test code in this repo. Maybe I am missing something ?

**Comment by BlackDex** [comment]
Well, there is, just run `cargo test --features sqlite,mysql,postgresql`
Also see https://github.com/dani-garcia/bitwarden_rs/blob/d69be7d03a0369faf1f6be6ed2cb908ec6b7a253/src/api/icons.rs#L141 as one example but there are more.

But, if you have time to help us add more useful tests, please feel free to do so.
